docs/articles_en/learn-openvino/llm_inference_guide/genai-guide-npu.rst

```diff
@@ -9,70 +9,114 @@ This guide will give you extra details on how to utilize NPU with the GenAI flav
 for information on how to start.
 
 Prerequisites
-#############
+#####################
 
 Install required dependencies:
 
 .. code-block:: console
 
    python -m venv npu-env
    npu-env\Scripts\activate
-   pip install optimum-intel nncf==2.11 onnx==1.16.1
+   pip install nncf==2.12 onnx==1.16.1 optimum-intel==1.19.0
    pip install --pre openvino openvino-tokenizers openvino-genai --extra-index-url https://storage.openvinotoolkit.org/simple/wheels/nightly
 
 Export an LLM model via Hugging Face Optimum-Intel
 ##################################################
 
-A chat-tuned TinyLlama model is used in this example. The following conversion & optimization
-settings are recommended when using the NPU:
+As **NPU supports only symmetrically-quantized 4-bit (INT4) models**, make sure to export
+the model with the proper conversion and optimization settings.
```

Comment:
I believe "only" is a strong statement, I'd call it "preferred"

Reply:
> I believe "only" is a strong statement, I'd call it "preferred"

```suggestion
Since **symmetrically-quantized 4-bit (INT4) models are preffered for inference on NPU**, make sure to export
the model with the proper conversion and optimization settings.
```